apps/oc_chef_wm/src/oc_chef_wm_named_policy.erl

```diff
@@ -0,0 +1,205 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Stephen Delano <stephen@chef.io>
+%% Copyright 2013-2014 Chef Software, Inc. All Rights Reserved.
+
+-module(oc_chef_wm_named_policy).
+
+-include("../../include/oc_chef_wm.hrl").
+
+%% Webmachine resource callbacks
+-mixin([{oc_chef_wm_base, [content_types_accepted/2,
+                           content_types_provided/2,
+                           finish_request/2,
+                           malformed_request/2,
+                           ping/2,
+                           forbidden/2,
+                           is_authorized/2,
+                           service_available/2]}]).
+
+-export([allowed_methods/2,
+         delete_resource/2,
+         from_json/2,
+         resource_exists/2,
+         to_json/2,
+         create_path/2]).
+
+%% chef_wm behavior callbacks
+-behaviour(chef_wm).
+-export([auth_info/2,
+         init/1,
+         init_resource_state/1,
+         malformed_request_message/3,
+         request_type/0,
+         validate_request/3,
+         conflict_message/1,
+         post_is_create/2]).
+
+-define(VALIDATION_CONSTRAINTS,
```

Comment:
We usually keep the validation rules in with the object module - any reason to change the pattern here?